scripts/partition_manager.py

```diff
@@ -20,8 +20,7 @@ def remove_item_not_in_list(list_to_remove_from, list_to_check):
 
 def item_is_placed(d, item, after_or_before):
     assert after_or_before in ['after', 'before']
-    return after_or_before in d['placement'].keys() and \
-           d['placement'][after_or_before][0] == item
+    return after_or_before in d['placement'].keys() and d['placement'][after_or_before][0] == item
```

Comment:
Seems like unnecessary diff ;). But while you are here:

```suggestion
    return after_or_before in d['placement'] and \
           d['placement'][after_or_before][0] == item
```

The expression `foo in some_dict` has the same value as `foo in some_dict.keys()`, but is a bit more idiomatic and avoids unnecessary allocation. Not sure if the same lint could be removed elsewhere in this code.

Reply:
Hehe, boy scout rule, I was in the neighborhood :) Agree, fixed.